schema/symptom.go

```diff
@@ -193,10 +194,9 @@ type SymptomReportData struct {
 	CustomizedSymptoms []Symptom `json:"customized_symptoms" bson:"customized_symptoms"`
 	Location           GeoJSON   `json:"location" bson:"location"`
 	Timestamp          int64     `json:"ts" bson:"ts"`
-	SymptomScore       float64   `json:"score" bson:"score"`
 }
 
-type SymptomDistribution map[SymptomType]int
+type SymptomDistribution map[string]int
```

Comment:
It is like the `SymptomType` is not welcomed. Do we consider to remove all of them and use `string` instead.

Reply:
Sounds good. My intention is to make the type of symptom ID as `string` instead of `SymptomType`. I can address this in another PR.